src/cdk/explorer/tree/treeInspector.ts

```diff
@@ -63,3 +63,19 @@ export function getDisplayLabel(construct: ConstructTreeEntity): string {
 
     return construct.id
 }
+
+
+/** 
+* (for CDK Visualization) Determines if a construct is a Resource and a state machine
+*
+* @param construct CDK construct
+*/
+export function isStateMachine(construct: ConstructTreeEntity): boolean{
```

Comment:
As we continue to add more code related to state machines, you may want to ask yourself whether it makes sense for this code to live in this file, or for it to be in its own file.